internal/cli/kraft/run/runner_kraftfile_runtime.go

```diff
@@ -358,6 +360,16 @@ func (runner *runnerKraftfileRuntime) Prepare(ctx context.Context, opts *RunOpti
 		machine.Spec.ApplicationArgs = runtime.Command()
 	}
 
+	var kernelArgs []string
+	if !runtime.KConfig().AllNoOrUnset(
+		"CONFIG_LIBUKRANDOM",
+		"CONFIG_LIBUKRANDOM_CMDLINE_INIT",
+	) && !(cpuid.CPU.Rdrand() && cpuid.CPU.Rdseed()) {
```

Comment:
There is a theoretical case where a CPU has `RDSEED` but not `RDRAND`, but I haven't come across such a thing in practice, so I'm not sure if it's worth adding a check. Saying that ,the opposite is possible: `RDRAND` was introduced before `RDSEED`, so there are CPUs out there that have the former but not the latter. The implementation in Unikraft has the minimum requirement of  `RDRAND` when the driver is enabled (seeding functions fall back to `RDRAND`), so in theory you may skip the check for `RDSEED` altogether.

Another thing related to portability: I had a quick look at https://github.com/klauspost/cpuid and it looks like it lacks detection for `FEAT_RNG` on Arm (along with many others). You may consider opening a PR upstream to add these so that they're there by the time we need them.

Reply:
noted, will do